src/useDarkMode.js

```diff
@@ -1,36 +1,35 @@
-import { useEffect } from "react";
-import useLocalStorage from "./useLocalStorage";
-import useMedia from "./useMedia";
+import { useEffect } from 'react';
+import useLocalStorage from './useLocalStorage';
+import useMedia from './useMedia';
 
 // Hook
 function useDarkMode() {
   // Use our useLocalStorage hook to persist state through a page refresh.
-  const [enabledState, setEnabledState] = useLocalStorage("dark-mode-enabled");
+  const [enabledState, setEnabledState] = useLocalStorage('dark-mode-enabled');
 
   // See if user has set a browser or OS preference for dark mode.
   const prefersDarkMode = useMedia(
-    ["(prefers-color-scheme: dark)"],
+    ['(prefers-color-scheme: dark)'],
     [true],
-    false
+    false,
   );
 
   // If enabledState is defined use it, otherwise fallback to prefersDarkMode.
   // This allows user to override OS level setting on our website.
-  const enabled =
-    typeof enabledState !== "undefined" ? enabledState : prefersDarkMode;
+  const enabled = typeof enabledState !== 'undefined' ? enabledState : prefersDarkMode;
```

Comment:
Why `typeof`?  just use its truthy value

Reply:
because you can set falsey values in localStorage